src/components/LazyLoadImage.jsx

```diff
@@ -73,8 +73,8 @@ class LazyLoadImage extends React.Component {
         className={wrapperClassName + ' lazy-load-image-background ' +
           effect + loadedClassName}
         style={{
-          backgroundImage: loaded ? '' : 'url( ' + placeholderSrc + ')',
-          backgroundSize: loaded ? '' : '100% 100%',
+          backgroundImage: loaded ? '' : (placeholderSrc ? 'url( ' + placeholderSrc + ')' : ''),
+          backgroundSize: loaded ? '' : (placeholderSrc ? '100% 100%' : ''),
```

Comment:
If I'm reading this correctly, this is equivalent, right?
```JSX
           backgroundImage: loaded || !placeholderSrc ? '' : 'url( ' + placeholderSrc + ')', 
           backgroundSize: loaded || !placeholderSrc ? '' : '100% 100%', 
```

Reply:
Yep, you are correct, I have updated the code, it is more readable that way Indeed @Aljullu , nice one